emanifest-py/pyproject.toml

```diff
@@ -1,6 +1,36 @@
+[project]
+name = "emanifest"
+version = "2.0.7"
+description = "An API utility wrapper for accessing the e-Manifest hazardous waste tracking system maintainedby the US Environmental Protection Agency"
+readme = "README.md"
+authors = [
+    {name = "William Nicholas ", email = "nicholas.william@epa.gov"},
+]
+maintainers = [
+    {name = "William Nicholas", email = "nicholas.william@epa.gov"},
+    {name = "David Graham", email = "graham.david@epa.gov"},
+]
+
+license = {text = "CCO 1.0"}
+requires-python = ">=3.7"
+dependencies = [
+    "requests==2.28.1",
+    "certifi==2022.12.7",
+    "requests-toolbelt==0.10.1",
+    "charset-normalizer==2.1.1",
+    "idna==3.4",
+    "urllib3==1.26.13"
+]
+
+[project.urls]
+issues = "https://github.com/USEPA/e-Manifest/issues"
+documentation = "https://github.com/USEPA/e-Manifest/emanifest-py"
+homepage = "https://github.com/USEPA/e-Manifest"
+repository = "https://github.com/USEPA/e-Manifest"
+
 [build-system]
 requires = [
     "setuptools>=42",
-    "wheel"
+    "wheel",
```

Comment:
Does this trailing comma (line 34, after "wheel") cause any problems?

Reply:
[trailing commas are A-OK in toml arrays](https://github.com/toml-lang/toml/issues/246#issuecomment-57919189) but I know they can bug people so if you want to adjust, I'm A-OK with that. I didn't have any problems building with the current condifs

Reply:
Doesn't bother me - if it works, no need to mess with it